packages/feedback-service/service.ts

```diff
@@ -37,27 +37,13 @@ mongoose.connection.on('error', error => {
 });
 
 /* Defining the Apollo Server */
-const apollo = new ApolloServer({
+const apollo = new ApolloServer( {
   playground: process.env.NODE_ENV !== 'production',
-  schema: mergeSchemas({
-    schemas: [
-      gqlSchema,
-    ],
-    resolvers: [
-      resolver,
-    ],
-  }),
-  subscriptions: {
-    path: '/subscriptions',
-  },
-  formatError: error => ({
-    message: error.message,
-    locations: error.locations,
-    stack: error.stack ? error.stack.split('\n') : [],
-    path: error.path,
-  }),
-  extensions
-});
+  schema: buildFederatedSchema( [ {
+    typeDefs: gqlSchema,
+    resolvers: resolver
+  }])});
```

Comment:
Why did you remove the rest of the options?
The `subscriptions` and `formatError`?

Reply:
As i seen in rigin email screenshot it was not there, i will revert them.

Reply:
@riginoommen  can you please confirm here as i can see in your email you sent there is no subscriptions and formatError ?

Reply:
@anjneeksharma That email was an example, so he only mentioned the part that needed to be changed. Read the email carefully. He specifically mentioned not to touch the other config files.
Also, the subscriptions and formatError is not related to the schemas.

Reply:
Also, the subscriptionsHandlers are being installed on line 63, to if you remove this, that line will probably throw an error.